packages/main/src/themes/AINotice.css

```diff
@@ -0,0 +1,6 @@
+.verificationText {
```

Comment:
we have convention for class names in css - look some of the existing compoenents